it/src/main/java/org/corfudb/universe/universe/vm/ApplianceManager.java

```diff
@@ -240,41 +245,6 @@ private VirtualMachineCloneSpec createLinuxCustomization(
         //Clone is powered on, not a template.
         vmCloneSpec.setPowerOn(true);
         vmCloneSpec.setTemplate(false);
-
```

Comment:
why this has removed?

Reply:
Because it is not needed. These options are defined by the template itself. In fact, vSphere refuses to clone the template if customization option are set.

Reply:
ok